### ISSUE TITLE
package.json minimum node version too low

### ISSUE THREAD

**Issue by neumann89** [cmake-js/cmake-js#50]
Hi,
I'm wanted to run cmake-js on a machine with Ubuntu 14.04 and legacy node version installed. The default version of the nodejs package is v0.10.25. In your package.json the minimum required version is: 
```
  "engines": {
    "node": ">= 0.10.0",
    "iojs": ">= 1.0.0"
  },
```  

But if I want to run `cmake-js compile` then I get the following error: 
```
.../node_modules/cmake-js/bin/cmake-js:140
for (var key of _.keys(argv)) {
             ^^
SyntaxError: Unexpected identifier
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3
```

On another machine with a newer node version cmake-js works perfect!

**Comment by unbornchikken** [comment]
It should work on node 0.10. I'm gonna fix it in the next release.

**Comment by unbornchikken** [comment]
Fixed as of 3.2.1.

**Comment by neumann89** [comment]
Thanks :+1:
